core/src/main/resources/dev/process_supported_files.py

```diff
@@ -0,0 +1,175 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Description:
+    This Python script goes over all directories of different spark versions under the
+    spark-rapids/tools/generated_files folder and contruct unions of the supported csv
+    files (supportedDataSource.csv, supportedExecs.csv, and supportedExprs.csv).
+
+Usage:
+    python process_supported_files.py path_to_generated_files_dir
+"""
+
+
+import os
+import argparse
+import pandas as pd
+from enum import IntEnum
+import logging
+
+
+"""
+IntEnum Class for support level types and easy comparison
+"""
+class SupportLevel(IntEnum):
+    NS = 1
+    NA = 2
+    PS = 3
+    S = 4
+    CO = 5
```

Comment:
CO is configured off, correct? So this takes precedence over `S` if there is an Exec with CO in one version and S in another Spark version?

Reply:
@nartal1 Thanks! I need to confirm what `CO` means. The current script assumes `CO` takes the highest precedence. I can change this once I get an answer.